contextual_lenses/tape_gfp/tape_gfp_utils.py

```diff
@@ -31,27 +33,24 @@ def dataset_to_df(in_name):
 
 
 # Padding.
-def pad_seq(seq, pad_char='-'):
```

Comment:
too much overlap with the pfam utils. Shouldn't you just have utils.py for this shared code? Otherwise, they may fall out of sync.

Reply:
pad_seq should not be here, the new method calls from protein_lm.domains, will look into this